exporters/metric/datadog/example_test.go

```diff
@@ -57,12 +59,15 @@ func ExampleExporter() {
 	go func() {
 		defer exp.Close()
 		processor := basic.New(selector, exp)
-		pusher := controller.New(processor, controller.WithExporter(exp), controller.WithCollectPeriod(time.Second*10))
+		pusher := controller.New(processor, controller.WithExporter(exp), controller.WithCollectPeriod(time.Second*10),
+			controller.WithResource(resource.Default()),
+			controller.WithResource(resource.NewSchemaless(semconv.ServiceNameKey.String("ExampleExporter"))))
```

Comment:
Should this use `New()` instead of `NewSchemaless()`?  We have a schema from `semconv.SchemaURL`.

Reply:
Thanks, @Aneurysm9

Okey! like this?

```go
ctx := context.Background()
serviceName, err := resource.New(ctx, resource.WithAttributes(semconv.ServiceNameKey.String("ExampleExporter")))
if err != nil {
	panic(err)
}

cont := controller.New(processor, controller.WithExporter(exp), controller.WithCollectPeriod(time.Second*10),
	controller.WithResource(resource.Default()), controller.WithResource(serviceName))
```